client/src/fix_kicker_imports.py

```
@@ -0,0 +1,54 @@
+    "module/student/roadmap/AiRoadmapWizardPage.tsx": "../../../components/ui/Kicker",
+    "module/student/roadmap/RoadmapDetailPage.tsx": "../../../components/ui/Kicker",
+    "module/student/roadmap/RoadmapEnrollPage.tsx": "../../../components/ui/Kicker",
+    "module/student/roadmap/RoadmapsLandingPage.tsx": "../../../components/ui/Kicker",
+}
+
+for rel_path, import_path in files.items():
+    with open(rel_path, "r", encoding="utf-8") as f:
+        lines = f.readlines()
+
+    # Remove ALL existing Kicker import lines (may be 2 due to previous runs)
+    lines = [l for l in lines if "import { Kicker }" not in l]
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)